geonetwork/transform-mcp-iso19115-3/schema/iso19115-3/convert/ISO19139/mapping/mcpsamplingfrequency.xsl

```diff
@@ -0,0 +1,79 @@
+                xmlns:mdb="http://standards.iso.org/iso/19115/-3/mdb/2.0"
+                xmlns:mdt="http://standards.iso.org/iso/19115/-3/mdt/1.0"
+                xmlns:mex="http://standards.iso.org/iso/19115/-3/mex/1.0"
+                xmlns:mic="http://standards.iso.org/iso/19115/-3/mic/1.0"
+                xmlns:mil="http://standards.iso.org/iso/19115/-3/mil/1.0"
+                xmlns:mrl="http://standards.iso.org/iso/19115/-3/mrl/2.0"
+                xmlns:mds="http://standards.iso.org/iso/19115/-3/mds/1.0"
+                xmlns:mmi="http://standards.iso.org/iso/19115/-3/mmi/1.0"
+                xmlns:mpc="http://standards.iso.org/iso/19115/-3/mpc/1.0"
+                xmlns:mrc="http://standards.iso.org/iso/19115/-3/mrc/1.0"
+                xmlns:mrd="http://standards.iso.org/iso/19115/-3/mrd/1.0"
+                xmlns:mri="http://standards.iso.org/iso/19115/-3/mri/1.0"
+                xmlns:mrs="http://standards.iso.org/iso/19115/-3/mrs/1.0"
+                xmlns:msr="http://standards.iso.org/iso/19115/-3/msr/2.0"
+                xmlns:mai="http://standards.iso.org/iso/19115/-3/mai/1.0"
+                xmlns:mdq="http://standards.iso.org/iso/19157/-2/mdq/1.0"
+                xmlns:gco="http://standards.iso.org/iso/19115/-3/gco/1.0"
+                xmlns:gml="http://www.opengis.net/gml/3.2"
+                xmlns:mcp="http://schemas.aodn.org.au/mcp-2.0"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl"
+                exclude-result-prefixes="#all">
+
+  <xsl:template match="mcp:samplingFrequency" mode="mcpsamplingfrequency">
+      <mri:temporalResolution>
+          <xsl:choose>
+            <xsl:when test="./gmd:MD_MaintenanceFrequencyCode[@codeListValue='hourly']">
+                <gco:TM_PeriodDuration>P0Y0M0DT1H0M0S</gco:TM_PeriodDuration>
+            </xsl:when>
+            <xsl:when test="./gmd:MD_MaintenanceFrequencyCode[@codeListValue='daily']">
+                <gco:TM_PeriodDuration>P0Y0M1DT0H0M0S</gco:TM_PeriodDuration>
+            </xsl:when>
+            <xsl:when test="./gmd:MD_MaintenanceFrequencyCode[@codeListValue='weekly']">
+                <gco:TM_PeriodDuration>P0Y0M7DT0H0M0S</gco:TM_PeriodDuration>
+            </xsl:when>
+            <xsl:when test="./gmd:MD_MaintenanceFrequencyCode[@codeListValue='fortnightly']">
+                <gco:TM_PeriodDuration>P0Y0M14DT0H0M0S</gco:TM_PeriodDuration>
+            </xsl:when>
+            <xsl:when test="./gmd:MD_MaintenanceFrequencyCode[@codeListValue='monthly']">
+                <gco:TM_PeriodDuration>P0Y1M0DT0H0M0S</gco:TM_PeriodDuration>
+            </xsl:when>
+            <xsl:when test="./gmd:MD_MaintenanceFrequencyCode[@codeListValue='quarterly']">
+                <gco:TM_PeriodDuration>P0Y4M0DT0H0M0S</gco:TM_PeriodDuration>
+            </xsl:when>
+            <xsl:when test="./gmd:MD_MaintenanceFrequencyCode[@codeListValue='biannually']">
```

Comment:
Biannualy is twice a year not every two years.

Reply:
Fixed.